mpf/devices/shot.py

```diff
@@ -412,6 +414,23 @@ def _delay_switch_hit(self, switch_name, **kwargs):
     def _release_delay(self, switch):
         self.active_delays.remove(switch)
 
+    def _register_jump_state_handlers(self):
+        for jump_state in self.config['jump_states']:
+            for event in jump_state['events']:
+                self._handlers.append(self.machine.events.add_handler(event, self._jump_states,
+                                                                      jump_state_config=jump_state))
+
+    def _remove_jump_state_handlers(self):
+        self.machine.events.remove_handlers_by_keys(self._handlers)
```

Comment:
The `_remove_switch_handlers()` method performs this same function and references the same `self._handlers` value, so this cleanup method will have no effect.

Reply:
Removed in latest commit.  Thanks for the review!